Examples/Gdip.Tutorial.11-Using.BRAs.to.perform.animations.ahk

```diff
@@ -0,0 +1,350 @@
+; gdi+ ahk tutorial 11 written by tic (Tariq Porter)
+; Requires Gdip.ahk either in your Lib folder as standard library or using #Include
+;
+; Tutorial to demonstrate how to use BRA files, in this instance for animation
+
+; #SingleInstance, Force
+; #NoEnv
+
+; Uncomment if Gdip.ahk is not in your standard library
+;#Include, Gdip.ahk
+
+; Start gdi+
+If !pToken := Gdip_Startup()
+{
+	MsgBox, 48, gdiplus error!, Gdiplus failed to start. Please ensure you have gdiplus on your system
+	ExitApp
+}
+; On exiting the program we will go to Exit subroutine to clean up any resources
+; OnExit, Exit
+OnExit("AppExit")  ; Requires [v1.1.20+]
+
+; I've added a simple new function here, just to ensure if anyone is having any problems then to make sure they are using the correct library version
+if (Gdip_LibraryVersion() < 1.30)
+{
+	MsgBox, 48, version error!, Please download the latest version of the gdi+ library
+	ExitApp
+}
+
+; Get the dimensions of the work area of the primary monitor
+MonInfo  := GetMonitorInfo(GetPrimaryMonitor())
+WALeft   := MonInfo.WALeft
+WATop    := MonInfo.WATop
+WARight  := MonInfo.WARight
+WABottom := MonInfo.WABottom
+WAWidth  := WARight - WALeft
+WAHeight := WABottom- WATop
+
+; Create our gui with WS_EX_LAYERED style = 0x80000
+Gui, 1: -Caption +E0x80000 +LastFound +OwnDialogs +Owner
+Gui, 1: Show, NA
+
+; Get a handle to this window
+hwnd1 := WinExist()
+
+; A BRA File is a collection of files packed into one Binary Resource Archive
+; There are a number of functions for use with BRA files. You must download the BRA Standard Library and either put it in your Lib folder or use #Include
+; The advantages over using the separate files are:
+; - One BRA file is cleaner to transport with your script than several other files
+; - It is easier to cleanup 1 BRA after use if needed
+; - A BRA file can be read once into memory using FileRead and is no longer needed
+; - A BRA file is better for your disk (especially flash disks) as it only uses 1 read operation. Flash disks have a finite number of read/writes
+; - BRA files are quicker to access as they are read from RAM and not from disk
+; - It is easier to use BRA files as the library provides all the functions needed
+
+; If the image we want to work with does not exist on disk, then download it...
+If !FileExist("Gdip.tutorial.file-fish.bra")
+{
+	MsgBox, 48, File error!, Please download Gdip.tutorial.file-fish.bra from`nwww.autohotkey.net/~tic/Gdip.tutorial.file-fish.bra
```

Comment:
`MsgBox, 48, File error!, Could not read file "Gdip.tutorial.file-fish.bra" in the current directory`